src/content/docs/infrastructure/infrastructure-monitoring/get-started/infrastructure-agent-eol-policy.mdx

```diff
@@ -1,16 +1,16 @@
 ---
 title: Infrastructure monitoring agent EOL policy
-metaDescription: Policies, start and end dates for support of New Relic infrastructure monitoring agent releases.
+metaDescription: Policies, start and end dates for support of New Relic infrastructure agent releases.
 freshnessValidatedDate: never
 ---
 
-The following are the specific policies and dates for support of our infrastructure monitoring agent. See our documentation about [general EOL policies](/docs/licenses/end-of-life/notification-changes-new-relic-saas-features-distributed-software/) for information about New Relic's overall end-of-life policy.
+The following are the specific policies and dates for support of our infrastructure agent. See our documentation about [general EOL policies](/docs/licenses/end-of-life/notification-changes-new-relic-saas-features-distributed-software/) for information about New Relic's overall end-of-life policy.
```

Comment:
Our infrastructure agent requires the following policies and dates

or something like that, just to remove the passive